web/package.json

```diff
@@ -103,7 +103,7 @@
     "babel-plugin-lodash": "^3.3.4",
     "babel-plugin-styled-components": "^1.10.0",
     "clean-webpack-plugin": "^3.0.0",
-    "copy-webpack-plugin": "^5.1.1",
+    "copy-webpack-plugin": "6.2.1",
```

Comment:
Does this already fix this issue: https://issues.tuerantuer.org/browse/IGAPP-481
@Znippet

Reply:
Yeah api changed and leads to this error. I fixed it, so issue should be solved. 
I updated to 6.2.1. cause 7.0.0 is not working properly yet.

Reply:
Okay I guess then we can close the issue and just upgrade further once the version is stable.